x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/logic/helpers.test.ts

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SchemaTypes } from '../../../../shared/types';
+
+import {
+  areFieldsAtDefaultSettings,
+  convertServerResultFieldsToResultFields,
+  convertToServerFieldResultSetting,
+  clearAllServerFields,
+  clearAllFields,
+  resetAllServerFields,
+  resetAllFields,
+  splitResultFields,
+} from './helpers';
+
```

Comment:
It looks like there isn't a specific unit test for `updateAllFields` - was that accidentally missed or are we incidentally testing it because it's reused by other helpers in the file? No worries if so, just wanted to check

Reply:
It is a private method, in this case, so I did not test it directly.